src/manifest_parser.h

```diff
@@ -31,10 +31,23 @@ enum DupeEdgeAction {
   kDupeEdgeActionError,
 };
 
+enum PhonyCycleAction {
+  kPhonyCycleActionWarn,
+  kPhonyCycleActionError,
+};
+
+struct ManifestParserOptions {
+  ManifestParserOptions():
+      dupe_edge_action_(kDupeEdgeActionWarn),
```

Comment:
Style like below is preferred.
```
  ManifestParserOptions()
      : dupe_edge_action_(kDupeEdgeActionWarn),
```
e.g. https://github.com/ninja-build/ninja/blob/master/src/graph.cc#L282